utils/scylla_doctor.py

```diff
@@ -114,8 +114,8 @@ def filter_out_failed_collectors(self, collector):
             return True
 
         # https://github.com/scylladb/scylladb/issues/18631
-        if collector in ["CPUSetCollector", "PerftuneSystemConfigurationCollector"]:
-            return True
+        # if self.node.distro.is_amazon2 and collector in ["CPUSetCollector", "PerftuneSystemConfigurationCollector"]:
```

Comment:
and this cause always to filter those collector, which wasn't the intention of this code part